Holovibes/sources/API.cc

```diff
@@ -1560,16 +1564,37 @@ std::optional<io_files::InputFrameFile*> import_file(const std::string& filename
 
 void set_input_file_start_index(size_t value)
 {
+    const bool is_data_moments = get_data_type() == RecordedDataType::MOMENTS;
+    // Ensures that moments are read 3 by 3
+    if (is_data_moments)
+        value -= value % 3;
+
     UPDATE_SETTING(InputFileStartIndex, value);
     if (value >= get_input_file_end_index())
-        set_input_file_end_index(value + 1);
+        set_input_file_end_index((value + is_data_moments) ? 3 : 1);
 }
 
 void set_input_file_end_index(size_t value)
 {
+    const bool is_data_moments = get_data_type() == RecordedDataType::MOMENTS;
+    // Ensures that moments are read 3 by 3
+    if (get_data_type() == RecordedDataType::MOMENTS)
+        value -= value % 3;
+
     UPDATE_SETTING(InputFileEndIndex, value);
     if (value <= get_input_file_start_index())
-        set_input_file_start_index(value - 1);
+        set_input_file_start_index(value - ((value + is_data_moments) ? 3 : 1));
+}
+
+void loaded_moments_data()
+{
+    set_batch_size(3);  // Cannot call the api.cc function because some parameters are not defined yet.
+    set_time_stride(3); // The user can change the time stride, but setting it to 3
+                        // is a good basis to analyze moments
+    close_windows();    // Set lens and raw view to false
```

Comment:
No need since when reading a file it will stop computation and close all windows